src/modules/bets.js

```diff
@@ -50,7 +87,9 @@ export default function bets(opts) {
         debug('enter', florins, wallet)
         if (wallet.florins < florins)
           return Promise.reject(new Error('You don\'t have that many florins.'))
-        return _entries[luser] = { user, option, florins }
+        db('bet_entries').where({ user }).del().catch(e => {throw e})
+        db('bet_entries').insert({ user, option, florins }).catch(e => {throw e})
```

Comment:
this seems like a possible race condition? (the insert might happen before the delete, since they're both async)